torchaudio/models/wav2vec2/model.py

```diff
@@ -8,9 +8,7 @@
 
 
 class Wav2Vec2Model(Module):
-    """torchaudio.models.Wav2Vec2Model(feature_extractor: torch.nn.Module, encoder: torch.nn.Module, aux: Optional[torch.nn.Module] = None)
```

Comment:
There are also instances of overriding function def for RNNT, SourceSeparationBundle, RNNTBundle, Wav2Vec2Bundle and Wav2Vec2ASRBundle, can those also be removed?

Reply:
Those overrides are different. They are there so as not to show the constructor signature, because Bundles are not meant to be instantiated by users.

Reply:
They will be handled (automated) using `:autosummary:`